setup.py

```diff
@@ -58,7 +55,7 @@
         'numpy >= 1.6',
         'OpenFisca-Core >= 0.5.0',
         'OpenFisca-Parsers >= 0.5',
-        'PasteDeploy',
```

Comment:
Gunicorn was used in prod for its compatibility with scipy (for the inversion revenu reform) because it is multi-process, whereas Paster is multithread